OptimizelySDK/OptlyConfig/OptimizelyConfig.cs

```diff
@@ -14,21 +14,28 @@
  * limitations under the License.
  */
 
+using Newtonsoft.Json;
 using System.Collections.Generic;
 
 namespace OptimizelySDK.OptlyConfig
 {
     public class OptimizelyConfig
     {
         public string Revision { get; private set; }
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public string SDKKey { get; private set; }
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public string EnvironmentKey { get; private set; }
         public IDictionary<string, OptimizelyExperiment> ExperimentsMap { get; private set; }
         public IDictionary<string, OptimizelyFeature> FeaturesMap { get; private set; }
 
         private string _datafile;
 
-        public OptimizelyConfig(string revision, IDictionary<string, OptimizelyExperiment> experimentsMap, IDictionary<string, OptimizelyFeature> featuresMap, string datafile = null)
+        public OptimizelyConfig(string revision, string sdkKey, string environmentKey, IDictionary<string, OptimizelyExperiment> experimentsMap, IDictionary<string, OptimizelyFeature> featuresMap, string datafile = null)
```

Comment:
This will break functionality. 

Add one more constructor and leave this constructor as it is.